docs/sphinx/manual/writing_single_node_hpx_applications.rst

```diff
@@ -749,23 +749,23 @@ Parallel algorithms
      * ``<hpx/include/uninitialized_value_construct.hpp>``
      * :cppreference-memory:`uninitialized_value_construct_n`
 
-.. list-table:: Index-based for-loops (In Header: `<hpx/include/parallel_algorithm.hpp>`)
+.. list-table:: Index-based for-loops (In Header: `<hpx/algorithm.hpp>`)
 
    * * Name
      * Description
      * In header
-   * * :cpp:func:`hpx::parallel::v2::for_loop`
+   * * :cpp:func:`hpx::for_loop`
      * Implements loop functionality over a range specified by integral or iterator bounds.
-     * ``<hpx/include/parallel_for_loop.hpp>``
-   * * :cpp:func:`hpx::parallel::v2::for_loop_strided`
+     * ``<hpx/algorithm.hpp>``
+   * * :cpp:func:`hpx::for_loop_strided`
      * Implements loop functionality over a range specified by integral or iterator bounds.
-     * ``<hpx/include/parallel_for_loop.hpp>``
-   * * :cpp:func:`hpx::parallel::v2::for_loop_n`
+     * ``<hpx/algorithm.hpp>``
+   * * :cpp:func:`hpx::for_loop_n`
      * Implements loop functionality over a range specified by integral or iterator bounds.
-     * ``<hpx/include/parallel_for_loop.hpp>``
-   * * :cpp:func:`hpx::parallel::v2::for_loop_n_strided`
+     * ``<hpx/algorithm.hpp>``
+   * * :cpp:func:`hpx::for_loop_n_strided`
```

Comment:
Thanks for changing it here as well... I think we have missed correcting all the other algorithms we have already ported to use CPOs here...

Reply:
See #4908.

Reply:
I know you already saw the PR, but for anyone else wondering about this: #4908.